Sources/StreamChatUI/ChatMessageList/ChatMessageListVC.swift

```diff
@@ -588,4 +556,35 @@ open class _ChatMessageListVC<ExtraData: ExtraDataTypes>:
         df.locale = .autoupdatingCurrent
         return df
     }()
+
+    // ======
+
+    open func numberOfSections(in tableView: UITableView) -> Int {
+        1
+    }
```

Comment:
This is not needed. System default is 1

Reply:
it's here to explicitly say the method is `open` so it can be overridden